lm_eval/tasks/mmlu_pro/README.md

```diff
@@ -57,3 +57,8 @@ If other tasks on this dataset are already supported:
 * [ ] Is the "Main" variant of this task clearly denoted?
 * [ ] Have you provided a short sentence in a README on what each new variant adds / evaluates?
 * [ ] Have you noted which, if any, published evaluation setups are matched by this variant?
+
+### Changelog
+
+* (tasks, group) 2024-09-23 -- (ver 1, ver 2)
```

Comment:
one nitpick: it's not super clear to me that `(ver 1, ver 2)` implies that this took us from version 1.0 to 2.0. perhaps `(version 1.0 --> version 2.0)` ?